unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/BuilderInWorldCommon/ToolTipController.cs

```diff
@@ -13,11 +13,19 @@ public class ToolTipController : MonoBehaviour
 
     Coroutine changeAlphaCor;
 
+
+    private void OnDestroy()
+    {
+        if (changeAlphaCor != null)
+            CoroutineStarter.Stop(changeAlphaCor);
+    }
+
+
     public void Desactivate()
```

Comment:
```suggestion
    public void Stop()
```
If possible let's use `Start` `Stop` to systems that have an ongoing `Coroutine`. We are trying to implement this naming to all our systems that behave like this.